libs-scala/gatling-utils/src/main/scala/com/daml/gatling/stats/FixTradeStats.scala

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.gatling.stats
+
+import com.daml.gatling.stats.SimulationLog.RequestTypeStats
+import scalaz._
+import Scalaz._
+
+// the actually-interesting statistics of a FixTrade simulation
+// We need to get these numbers out of a json file produced by the file writer.
```

Comment:
This comment is stale (reading the json file from the html report was the previous implementation).
And, as mentioned, this file should be replaced by one that suits your use case(s).